core/tests.py

```diff
@@ -1,3 +1,11 @@
 from django.test import TestCase
-
+from core.models import *
 # Create your tests here.
```

Comment:
Are all the tests for the entire project included here? Or are there multiple files?